scripts/createtree.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -eu
+
+if [ $# -ne 2 ]; then
+    echo "Usage: $0 <treeid> <treetype>"
+    exit 1
+fi
+
+testdbopts='-u test --password=zaphod -D test'
+tree_id=$1
+tree_type=$2
+
+mysql ${testdbopts} -e "
+    INSERT INTO Trees(TreeId, TreeState, TreeType, HashStrategy, HashAlgorithm, SignatureAlgorithm, DuplicatePolicy, CreateTime, UpdateTime)
+    VALUES (${tree_id}, 'ACTIVE', '${tree_type}', 'RFC_6962', 'SHA256', 'RSA', 'NOT_ALLOWED', NOW(), NOW())"
+mysql ${testdbopts} -e "
+    INSERT INTO TreeControl(TreeId, SigningEnabled, SequencingEnabled, SequenceIntervalSeconds, SignIntervalSeconds)
+    VALUES(${tree_id}, false, false, 1, 1)"
```

Comment:
Are we going to need `true, true` for `tree_type=LOG` at some point?  (Possibly taken care of by the scripts triggering the admin code rather than manually fiddling with SQL..)